sdk/communication/communication-chat/src/chatThreadClient.ts

```diff
@@ -61,8 +70,8 @@ export class ChatThreadClient {
   private timeOfLastTypingRequest: Date | undefined = undefined;
 
   constructor(
-    threadId: string,
     private readonly url: string,
+    threadId: string,
```

Comment:
I remember that one architect suggested to have a single endpoint argument which is <resource/threads/{threadId}>. I don't remember what the conclusion was.

@juancamilor

Reply:
based on our review last week, I think we got the agreement with these properties. Just made the threadId the second param